addons/hr_skills_survey/models/hr_resume_line.py

```diff
@@ -2,7 +2,7 @@
 
 from dateutil.relativedelta import relativedelta
 
-from odoo import api, fields, models
+from odoo import api, fields, models, _
```

Comment:
you do not need this anymore
```suggestion
from odoo import api, fields, models
```